packages/samples/search-app-router-nextjs/src/components/atomic-search-page.tsx

```diff
@@ -0,0 +1,233 @@
+'use client';
+
+import {
+  AtomicBreadbox,
+  AtomicColorFacet,
+  AtomicDidYouMean,
+  AtomicFacet,
+  AtomicFacetManager,
+  AtomicFormatCurrency,
+  AtomicLayoutSection,
+  AtomicLoadMoreResults,
+  AtomicNoResults,
+  AtomicNumericFacet,
+  AtomicNumericRange,
+  AtomicQueryError,
+  AtomicQuerySummary,
+  AtomicRatingFacet,
+  AtomicRatingRangeFacet,
+  AtomicRefineToggle,
+  AtomicResultBadge,
+  AtomicResultDate,
+  AtomicResultFieldsList,
+  AtomicResultImage,
+  AtomicResultLink,
+  AtomicResultList,
+  AtomicResultMultiValueText,
+  AtomicResultNumber,
+  AtomicResultPrintableUri,
+  AtomicResultRating,
+  AtomicResultSectionBadges,
+  AtomicResultSectionBottomMetadata,
+  AtomicResultSectionEmphasized,
+  AtomicResultSectionExcerpt,
+  AtomicResultSectionTitle,
+  AtomicResultSectionTitleMetadata,
+  AtomicResultSectionVisual,
+  AtomicResultText,
+  AtomicSearchBox,
+  AtomicSearchInterface,
+  AtomicSearchLayout,
+  AtomicSortDropdown,
+  AtomicSortExpression,
+  AtomicText,
+  AtomicTimeframe,
+  AtomicTimeframeFacet,
+} from '@coveo/atomic-react';
+import {buildSearchEngine, type Result} from '@coveo/headless';
+import type {NextPage} from 'next';
+import {useMemo} from 'react';
+
+const SearchPage: NextPage = () => {
+  const engine = useMemo(
+    () =>
+      buildSearchEngine({
+        configuration: {
+          accessToken: 'xxc23ce82a-3733-496e-b37e-9736168c4fd9',
+          organizationId: 'electronicscoveodemocomo0n2fu8v',
```

Comment:
Why not use the searchuisamples org with non-commerce data?

In general, I think we should avoid showcasing commerce implementations build with the search engine in our samples. Commerce implementations should use headless commerce or atomic commerce.